src/main/java/de/tum/in/www1/artemis/domain/ComplaintResponse.java

```diff
@@ -17,22 +22,61 @@
 @Table(name = "complaint_response")
 @Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
 @JsonInclude(JsonInclude.Include.NON_EMPTY)
-public class ComplaintResponse extends DomainObject {
+public class ComplaintResponse extends AbstractAuditingEntity {
```

Comment:
Looking at the usages again, we could also just keep the `createdTime` and make it `@CreatedDate`, this could also directly be `ZonedDateTime`, as far as I see. But in the end it might not matter as much.

Reply:
Using AbstractAuditingEntity was decided in discussion with @krusche as it lowers the risk that somebody forgets to set the created date

Reply:
I added a comment to clarify in acca9065ac91e5e835d914cbbefdbf9c77af9faf